internal/cmd/providers.go

```diff
@@ -92,13 +104,14 @@ $ infra providers edit okta --client-secret VT_oXtkEDaT7UFY-C3DSRWYb00qyKZ1K1VCq
 # Connect Google to Infra with group sync
 $ infra providers edit google --client-secret VT_oXtkEDaT7UFY-C3DSRWYb00qyKZ1K1VCq7YzN --service-account-key ~/client-123.json --service-account-email hello@example.com --workspace-domain-admin admin@example.com
 `,
-		Args: ExactArgs(1),
+		Args: MaxArgs(1),
```

Comment:
This needs to be `ExactArgs`, right? Because we use `args[0]` unconditionally, so no args will cause a panic.